src/main/java/edu/utdallas/davisbase/parser/ParseWhereException.java

```diff
@@ -0,0 +1,25 @@
+package edu.utdallas.davisbase.parser;
+
+import edu.utdallas.davisbase.DavisBaseException;
+
+public class ParseWhereException extends DavisBaseException {
```

Comment:
`ParseWhereException` should extend `ParseException` so that the main driver can group all types of parsing errors with a single `catch (ParseException e)` statement.